content/docs/use-cases/experiment-bookkeeping.md

```diff
@@ -0,0 +1,32 @@
+# Data Science Experiment Bookkeeping
+
+Mature data science required constant research and improvement. This implies
+continuous experimentation on any stage of the data lifecycle, from acquisition
+to processing and usage, for example to retrain machine learning models for
+better accuracy. Keeping track of all these experiments is challenging, not to
+mention being able to find and reproduce them again in the future.
+
+The core of DVC lets you codify your project's
```

Comment:
I don't think we need to mention data pipelines at all - it complicates the story. Mention exp management. Mention after that - that it includes data pipelines ...

Reply:
Hi @shcheklein FYI I've pushed up to here so far. Shooting to address most of this feedback by our call in a little bit. Thanks